compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -330,7 +330,8 @@ internal class RootNodeOwner(
         ): Nothing {
             awaitCancellation()
         }
-        override val dragAndDropManager: DragAndDropManager = this@RootNodeOwner.dragAndDropManager
+        override val dragAndDropManager: DragAndDropManager =
+            this@RootNodeOwner.dragAndDropManager.asDragAndDropManager()
```

Comment:
You're creating a new object each time now

Reply:
What do you mean by "time"?

Reply:
Every property get call

Reply:
No, that would happen if it was e.g.
```
override val dragAndDropManager: DragAndDropManager =
    get() = this@RootNodeOwner.dragAndDropManager.asDragAndDropManager()
```

Reply:
Ah, right